### PR TITLE
Mate-Rov to MTS-AUVZHCET

### LINKED ISSUE #39: Rename MATE-ROV as MTS-AUVZHCET

### PR DESCRIPTION
**Pull request by shubh2001**
fix #39

### CODE REVIEW

**Comment by pnijhara** [comment]
So basically, the first paragraph describes what MTS is, but 2nd paragraph describes ROV competition which is of no use. So we need to replace it with MTS-AUVZHCET, that how the club became part of this org and what is its responsibilities now. You can ask the management team for further help.

**Comment by shubh2001** [comment]
Okay I removed that paragraph as well. Will look for content from the management team.

**Comment by pnijhara** [comment]
@shubh2001, any updates on this?

**Comment by shubh2001** [comment]
I've asked Salma di for the content. Will update as soon as I get it.

**Comment by shubh2001** [comment]
Added @pnijhara

File: mate_rov.html [hunk 1]
```diff
@@ -222,27 +222,26 @@ <h2 class="title">About MATE ROV</h2>
...
+							  world as <strong>"What we have out there is all we have; there is no more"</strong>.</p>
+						<p class="about-des">And this is where your mission begins.
+							<br>
+							Task1: The Ubiquitous Problem of Plastic Pollution
+							<br>
+							Task2: The Catastrophic Impact of Climate Change on Coral Reefs
+							<br>
+							Task 3: Maintaining Healthy Waterways Part II: Delaware River and Bay
+							<br><br>
+						</p>
```
Comment (pnijhara) [inline mate_rov.html:244]: Remove this also.
And change the file name from `mate_rov.html` to `mts_auvzhcet.html`

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: mate_rov.html (hunk 1 above)
Comment (shubh2001) [reply mate_rov.html]: done

**Comment by pnijhara** [comment]
Hey @shubh2001 please revert the file's name changing commit. This is giving merge conflicts. We can rename the file in some other PR.
Apologies!

**Comment by shubh2001** [comment]
@pnijhara bhai I hope its okay now.

**Review by pnijhara — APPROVED** [review]
Good to go
Thanks, @shubh2001

**Pull request merged**